fusor/models.py

```diff
@@ -609,12 +566,18 @@ def enforce_element_quantities(cls, values):
             raise ValueError(uq_gene_msg)
         return values
 
-    @root_validator(skip_on_failure=True)
+    @model_validator(mode="after")
```

Comment:
should this be `before` like the others? I wasn't sure

Reply:
https://docs.pydantic.dev/latest/concepts/validators/#before-after-wrap-and-plain-validators